File: noobs/q4/solution.py
def smallest(a,b,c):
    if(a<b and a<c):
        return a
    elif(b<a and b<c):
        return b
    elif(c<a and c<b):
        return c
    else:
        print("All are same")

x = 0
y = 0
z = 0
result = smallest(x,y,z)
print(result)

File: noobs/q4/solution.cpp
#include<iostream>
using namespace std;

int smallest(int a,int b,int c){
    if(a<b && a<c){
        return a;
    } else if(b<a && b<c){
        return b;
    } else if(c<a & c<a){
        return c;
    } else{
        cout << "All numebers are same" << endl;
    }
    return 0;
}

int main(){
    int x = 5;
    int y = 4;
    int z = 3;
    cout << smallest(x,y,z) << endl;

    return 0;
}